src/components/Common/Charts/ObservationChart.tsx

```
@@ -35,19 +35,10 @@ import { Avatar } from "@/components/Common/Avatar";
 import query from "@/Utils/request/query";
 import { formatName } from "@/Utils/utils";
 import { Code } from "@/types/base/code/code";
-import patientApi from "@/types/emr/patient/patientApi";
 
+import { formatName } from "@/Utils/utils";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'formatName'. Consider to delete it or rename it.

'formatName' is defined here:


(lint/suspicious/noRedeclare)